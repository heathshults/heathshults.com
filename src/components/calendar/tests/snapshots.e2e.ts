import { snapIt } from '../../../test';

const component = 'calendar';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap('a default calendar', '<hs-calendar date="1982, January 14"></hs-calendar>');
    snap('a coloured calendar', '<hs-calendar type="info" date="1982, January 14"></hs-calendar>');
  });
});
