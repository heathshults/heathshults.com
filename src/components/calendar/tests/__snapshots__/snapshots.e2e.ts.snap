// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`calendar renders a coloured calendar 1`] = `"<hs-calendar type=\\"info\\" date=\\"1982, January 14\\" class=\\"hydrated\\"><div class=\\"hs-calendar\\"><button class=\\"hs-calendar__control hs-calendar__control--prev-year\\">‹</button><div class=\\"hs-calendar__header hs-calendar__header--year\\">1982</div><button class=\\"hs-calendar__control hs-calendar__control--next-year\\">›</button><button class=\\"hs-calendar__control hs-calendar__control--prev-month\\">‹</button><div class=\\"hs-calendar__header hs-calendar__header--month\\">January</div><button class=\\"hs-calendar__control hs-calendar__control--next-month\\">›</button><div class=\\"hs-calendar__day\\">Su</div><div class=\\"hs-calendar__day\\">Mo</div><div class=\\"hs-calendar__day\\">Tu</div><div class=\\"hs-calendar__day\\">We</div><div class=\\"hs-calendar__day\\">Th</div><div class=\\"hs-calendar__day\\">Fr</div><div class=\\"hs-calendar__day\\">Sa</div><button class=\\"hs-calendar__date hs-button\\" aria-selected=\\"0\\">27</button><button class=\\"hs-calendar__date hs-button\\" aria-selected=\\"0\\">28</button><button class=\\"hs-calendar__date hs-button\\" aria-selected=\\"0\\">29</button><button class=\\"hs-calendar__date hs-button\\" aria-selected=\\"0\\">30</button><button class=\\"hs-calendar__date hs-button\\" aria-selected=\\"0\\">31</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">1</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">2</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">3</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">4</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">5</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">6</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">7</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">8</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">9</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">10</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">11</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">12</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">13</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button hs-button--info\\" aria-selected=\\"1\\">14</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">15</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">16</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">17</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">18</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">19</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">20</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">21</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">22</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">23</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">24</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">25</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">26</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">27</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">28</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">29</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">30</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">31</button><button class=\\"hs-calendar__date hs-button\\" aria-selected=\\"0\\">1</button><button class=\\"hs-calendar__date hs-button\\" aria-selected=\\"0\\">2</button><button class=\\"hs-calendar__date hs-button\\" aria-selected=\\"0\\">3</button><button class=\\"hs-calendar__date hs-button\\" aria-selected=\\"0\\">4</button><button class=\\"hs-calendar__date hs-button\\" aria-selected=\\"0\\">5</button><button class=\\"hs-calendar__date hs-button\\" aria-selected=\\"0\\">6</button><div class=\\"hs-calendar__footer\\"><button class=\\"hs-calendar__today hs-button hs-button--block hs-button--info\\">Today</button></div></div></hs-calendar>"`;

exports[`calendar renders a default calendar 1`] = `"<hs-calendar date=\\"1982, January 14\\" class=\\"hydrated\\"><div class=\\"hs-calendar\\"><button class=\\"hs-calendar__control hs-calendar__control--prev-year\\">‹</button><div class=\\"hs-calendar__header hs-calendar__header--year\\">1982</div><button class=\\"hs-calendar__control hs-calendar__control--next-year\\">›</button><button class=\\"hs-calendar__control hs-calendar__control--prev-month\\">‹</button><div class=\\"hs-calendar__header hs-calendar__header--month\\">January</div><button class=\\"hs-calendar__control hs-calendar__control--next-month\\">›</button><div class=\\"hs-calendar__day\\">Su</div><div class=\\"hs-calendar__day\\">Mo</div><div class=\\"hs-calendar__day\\">Tu</div><div class=\\"hs-calendar__day\\">We</div><div class=\\"hs-calendar__day\\">Th</div><div class=\\"hs-calendar__day\\">Fr</div><div class=\\"hs-calendar__day\\">Sa</div><button class=\\"hs-calendar__date hs-button\\" aria-selected=\\"0\\">27</button><button class=\\"hs-calendar__date hs-button\\" aria-selected=\\"0\\">28</button><button class=\\"hs-calendar__date hs-button\\" aria-selected=\\"0\\">29</button><button class=\\"hs-calendar__date hs-button\\" aria-selected=\\"0\\">30</button><button class=\\"hs-calendar__date hs-button\\" aria-selected=\\"0\\">31</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">1</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">2</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">3</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">4</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">5</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">6</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">7</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">8</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">9</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">10</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">11</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">12</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">13</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button hs-button--brand\\" aria-selected=\\"1\\">14</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">15</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">16</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">17</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">18</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">19</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">20</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">21</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">22</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">23</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">24</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">25</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">26</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">27</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">28</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">29</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">30</button><button class=\\"hs-calendar__date hs-calendar__date--in-month hs-button\\" aria-selected=\\"0\\">31</button><button class=\\"hs-calendar__date hs-button\\" aria-selected=\\"0\\">1</button><button class=\\"hs-calendar__date hs-button\\" aria-selected=\\"0\\">2</button><button class=\\"hs-calendar__date hs-button\\" aria-selected=\\"0\\">3</button><button class=\\"hs-calendar__date hs-button\\" aria-selected=\\"0\\">4</button><button class=\\"hs-calendar__date hs-button\\" aria-selected=\\"0\\">5</button><button class=\\"hs-calendar__date hs-button\\" aria-selected=\\"0\\">6</button><div class=\\"hs-calendar__footer\\"><button class=\\"hs-calendar__today hs-button hs-button--block hs-button--brand\\">Today</button></div></div></hs-calendar>"`;
